src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck has started working.");
```

Comment:
The message 'Truck has started working.' could be more informative. According to the checklist, it should indicate the type of `Machine` that is working. Consider specifying that this is a Truck, for example: 'Truck started its engine and began to work.'